src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
```

Comment:
what is this rule for?
I checked and it works without this rule
![image](https://github.com/mate-academy/layout_stars/assets/43861663/c291c0c0-773e-4cb8-9283-788c9e1077a7)